SignalMessaging/contacts/SelectThreadViewController.m

```diff
@@ -116,8 +116,7 @@ - (void)createViews
     [self.view addSubview:self.tableViewController.view];
     [_tableViewController.view autoPinWidthToSuperview];
     [_tableViewController.view autoPinEdge:ALEdgeTop toEdge:ALEdgeBottom ofView:header];
-
-    [self autoPinViewToBottomGuideOrKeyboard:self.tableViewController.view];
+    [_tableViewController.view autoPinEdgeToSuperviewEdge:ALEdgeBottom];
```

Comment:
I attempted to follow the behavior of the native iOS contacts app WRT table contents flowing below the iPhoneX button.